acceptable/__main__.py

```diff
@@ -153,6 +153,19 @@ def parse_args(raw_args=None, parser_cls=None, stdin=None):
 
     lint_parser.set_defaults(func=lint_cmd)
 
+    doubles_parser = subparser.add_parser(
+        'doubles',
+        help='Generate test doubles'
+    )
+    doubles_parser.add_argument(
+        'metadata',
+        nargs='?',
+        type=argparse.FileType('r'),
+        default=stdin,
+        help='metadata file path, uses stdin if omitted'
+    )
```

Comment:
Should we add an argument for output file too?

Reply:
The metadata command doesn't have one so I feel its better to be consistent. If we want we could add an output param to the metadata and double commands later.